tests/unit/test_mongokafka.py

```diff
@@ -15,8 +15,15 @@
 """Unit tests related to the MongoKafka functionality."""
 
 import logging
+from unittest.mock import AsyncMock
 
+import pytest
+from pydantic import BaseModel
+
+from hexkit.correlation import set_new_correlation_id
+from hexkit.protocols.dao import DbTimeoutError
 from hexkit.providers.mongokafka import MongoKafkaConfig
+from hexkit.providers.mongokafka.provider import MongoKafkaDaoPublisherFactory
 
 
 def make_mongokafka_config(kafka_max_message_size: int = 1048576) -> MongoKafkaConfig:
```

Comment:
You should mention here that the mongo DSN is supposed to *not* be running.

Reply:
To the `make_mongokafka_config` function.

Reply:
Oh I see now. The way it captured the import addition threw me off. See [89c5d87](https://github.com/ghga-de/hexkit/pull/132/commits/89c5d870b72b8c52bbba90e96a5069c8cb2e877f) for the docstring update